script.js

```diff
@@ -0,0 +1,123 @@
+	// Script
+    document.addEventListener('DOMContentLoaded', () => {
+        // Get the wrapper.
+            const block = document.querySelector('.q-and-a');
+    
+        // Get the video wrapper.
+            const video = block.querySelector('.q-and-a__video');
```

Comment:
suggestion: block could return null and if that were to happen, the video const would throw an error. Best to setup a gate to check for its existence before making an assumption.

Reply:
It may be good to add a gated check overall, and avoid any further execution. Since you assume block exists in a few places, this may be the best approach.

eg.
```
if (!block) {
    return;
}
```

Reply:
This is in my code. I left it out of the codepen for the sake of decluttering to display the issue I was facing with the button progress.